index.html

```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
       </div>
     </section>
 
+    <section class="contacts-area">
+      <div class="contacts">
+        <h6>CONTACT US</h6>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form class="contacts-form" action="#" method="post">
+        <ul class="input-area">
+          <li class="text-area">
+            <label for="name"></label>
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Text Input"
+            />
```

Comment:
- [ ] Kindly add the validation for the name input field, and add the `maxlength="30"` attribute, which checks the length of the input and ensures it's not beyond 30 characters.